package.json

```diff
@@ -3,19 +3,24 @@
   "version": "0.1.0",
   "description": "A class designed to enable easily extending the built-in javascript Error, allowing typed errors.",
   "main": "src/typed-error.js",
+  "types": "src/typed-error.ts",
   "scripts": {
-    "test": "node node_modules/coffee-script/bin/coffee -c . && node node_modules/mocha/bin/mocha"
+    "prepublish": "npm test",
```

Comment:
`prepublish` is deprecated, this should probably be prepublishOnly nowadays: http://blog.lholmquist.org/npm-prepublish-changes/